pkg/common/model.go

```diff
@@ -214,14 +214,14 @@ type GlobalSearchResultResp struct {
 }
 
 type GlobalSearchResult struct {
```

Comment:
Don't these renames mean we break backwards compatibility with older zots? Do we care about that?

Reply:
From what I gathered, the GlobalSearch and other such payloads that are part of the GQL schema are handled entirely by the GQL server.

Ideally, this implementation should send the same payload as the GQL server except aggregated.

Not sure if I fully got your question though.

Reply:
So you mean this code did not produce the same payload as the GQL server before the change?
In this case we should definitely fix this issue.